core/kazoo_config/src/kapps_config_util.erl

```diff
@@ -49,9 +49,9 @@ load_config_from_reseller(Account, Config) ->
     ResellerId = kz_services:find_reseller_id(Account),
     load_config_from_account(ResellerId, Config).
 
--spec load_config_from_system(api_binary(), ne_binary()) -> {ok, kz_json:object()} | {error, any()}.
+-spec load_config_from_system(api_binary(), ne_binary()) -> {ok, kz_json:object()}.
 load_config_from_system(_Account, Config) ->
-    {'ok', kz_json:get_value(<<"default">>, maybe_new(kapps_config:get_category(Config)))}.
+    {'ok', kz_json:get_value(<<"default">>, kapps_config:get_category(Config), kz_json:new())}.
```

Comment:
account_config.paypal.json  system_config.paypal.json are both added.

If I mistype paypal, I'll get an error anyway since there will be no schemas with such a typo...

I don't insist here at all. If such 'let it crash' thing was added by intention, I don't mind just to add doc to system_config db, but I doubt it was intended..